tests/legacy-cli/e2e/tests/commands/config/config-global-completion.ts

```diff
@@ -0,0 +1,18 @@
+import { homedir } from 'os';
+import * as path from 'path';
+import { deleteFile, expectFileToExist } from '../../../utils/fs';
+import { ng } from '../../../utils/process';
+import { expectToFail } from '../../../utils/utils';
+
+export default async function () {
+  await expectToFail(() => ng('config', 'cli.completion.prompted', 'true'));
+  await expectToFail(() => ng('config', 'cli.completion.prompted'));
+
+  await ng('config', '--global', 'cli.completion.prompted', 'true');
+  const { stdout } = await ng('config', '--global', 'cli.completion.prompted');
+  if (!stdout.includes('true')) {
+    throw new Error(`Expected "true", received "${JSON.stringify(stdout)}".`);
+  }
```

Comment:
Can you please add  a test that checks that no additional properties can be added

Example: 
```ts
const { message } = await expectToFail(() => silentNg('config', '--global', 'cli.completion.invalid', 'true'));

const { message } = await expectToFail(() => silentNg('config', '--global', 'cli.cache.enabled','true'));
```